addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -78,27 +80,63 @@ <H2>Binary Incompatible Changes</H2>
 The following methods (all of which were deprecated more than 5 years ago) have been removed:
 
 <ul>
+<li><code>org.parosproxy.paros.control.Control#createAndOpenUntitledDb()</code></li>
 <li><code>org.parosproxy.paros.core.scanner.Alert#getReliability()</code></li>
+<li><code>org.parosproxy.paros.core.scanner.Alert#getAlert()</code></li>
+<li><code>org.parosproxy.paros.core.scanner.Alert#setAlert(String)</code></li>
 <li><code>org.parosproxy.paros.core.scanner.Alert#setDetail(String, String, String, String, String, String, String, HttpMessage) </code></li>
 <li><code>org.parosproxy.paros.core.scanner.Alert#setRiskReliability(int, int)</code></li>
+<li><code>org.parosproxy.paros.core.scanner.HostProcess#setPluginRequestCount(int,int)</code></li>
+<li><code>org.parosproxy.paros.core.scanner.VariantAbstractQuery#setParams(int,java.util.map)</code></li>
+<code><li>org.parosproxy.paros.core.scanner.HostProcess#setTestCurrentCount(org.parosproxy.paros.core.scanner.Plugin,int)</code></li>
```

Comment:
- Need to be reordered.
- In the last line the opening tags are not in the correct order.